src/main/java/zmaster587/advancedRocketry/dimension/DimensionManager.java

```diff
@@ -372,10 +372,28 @@ public DimensionProperties generateRandomGasGiant(int starId, String name, int b
 		properties.averageTemperature = AstronomicalBodyHelper.getAverageTemperature(properties.getStar(), properties.getSolarOrbitalDistance(), properties.getAtmosphereDensity());
 		properties.setGasGiant(true);
 
-		// Add all gasses for the default world
-		// TODO: add variation
-		for( Fluid gas : AdvancedRocketryFluids.getGasGiantGasses() )
-			properties.getHarvestableGasses().add(gas);
+		// Add gasses to gas giants, for Jupiter-like planets add hydrogen (and helium and helium-3 if they exist) and for ice giants add nitrogen and oxygen, with ammonia and methane if they exist
+		if(properties.getGravitationalMultiplier() > 1.25){
+			properties.getHarvestableGasses().add(AdvancedRocketryFluids.fluidHydrogen);
+			if (FluidRegistry.isFluidRegistered("helium")){
```

Comment:
We should probably be checking the `AdvancedRocketryFluids` class to see if it's registered there rather than the forge fluid registry.  Since we check to see if the forge registry has the fluid already before adding it to the  `AdvancedRocketryFluids` registry, we don't need to worry about it not being registered in forge.

Since the Ar fluid registry will later be configurable, it may not have some fluids registered that are in forge.  It think it'd be easier to just add that now than later since it should be a very easy change.